src/js/common/components/CopyTextToClipboardButton/CopyTextToClipboardButton.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react';
+
+import { IconButton, Tooltip } from '@material-ui/core';
+import { Done, FileCopyOutlined } from '@material-ui/icons';
+import PropTypes from 'prop-types';
+import { useTranslation } from 'react-i18next';
+
+const CopyTextToClipboardButton = ({ textToCopy }) => {
+  const [isShowingSuccessTooltip, setIsShowingSuccessTooltip] = useState(false);
+
+  const { t } = useTranslation('common');
+
+  const copyToClipboard = () => {
+    navigator.clipboard.writeText(textToCopy);
```

Comment:
This line returns a Promise. I don't know if the operation could be slow or throw an error in some cases. I think you can use `async await` here and wrap the function body with a `try catch` block. In the **catch block** you can just do this: `setIsShowingSuccessTooltip(false)`